holmes/core/llm.py

```diff
@@ -192,6 +192,7 @@ def completion(
         thinking = None
         if THINKING:  # if model requires 'thinking', load it from env vars
             thinking = json.loads(THINKING)
+            litellm.modify_params = True
```

Comment:
Why do we need this? Tried to understand from the docs but not sure i was able to.

Reply:
This THINKING and this one are for Anthropic
It needs it, otherwise there's an error with some of the tools